samples/nhibernate/custom-mappings/sample.md

```diff
@@ -1,38 +1,33 @@
 ---
 title: Custom mapping
 summary: Create a custom mapping to change how NHibernate creates the database schema using different techniques.
-reviewed: 2016-03-21
+reviewed: 2018-01-29
 component: NHibernate
 related:
  - persistence/nhibernate
 ---
 
 
-Sometimes the database schema that gets generated by `NServiceBus.NHibernate` for the saga data types is insufficient. For example, storing strings with a different string length, want so use complex types, need to store DateTime values with a high precision or maybe want to tweak the eager versus lazy loading to improve performance. A custom mapping will be required to achieve this, so that the generated default mapping will not be used.
+Sometimes the database schema that is generated by `NServiceBus.NHibernate` is insufficient for saga data types. For example, storing strings with a different string length, wanting to use complex types, needing to store DateTime values with a high precision, or wanting to tweak the eager versus lazy loading rules might be better options. A custom mapping will be required to achieve these changes.
 
-Custom mapping options:
+Custom mapping can be added by:
 
- * Create a `*.hbm.xml` file for the saga data class
- * Use Fluent NHibernate (popular separate fluent API)
- * Use NHibernate.Mapping.Attributes
+ * Creating a `*.hbm.xml` file for the saga data class
+ * Fluent NHibernate (popular separate fluent API)
+ * NHibernate.Mapping.Attributes
  * Loquacious Configuration (native fluent API)
 
-There are probably even other options, but these are the most frequently used.
+There are other options, but these are the most frequently used. The performance difference between these options can hae small variances, but those will occur during endpoint startup, not when the endpoint is processing messages.
 
-Which method to use depends on if mapping as xml, fluent code or code attributes. There can be small performance differences during startup but not when processing messages.
+NOTE: There is no requirement to create a custom mapping for all sagas if a custom mapping is needed by only one saga. The NHibernate configuration in each sample type demonstrate how both generated and custom mappings can work simultaneously.
```

Comment:
"in each sample type demonstrate how..." ?

Reply:
The 2nd sentence doesn't really add anything so I removed it completely